lemur/plugins/lemur_gcp/certificates.py

```diff
@@ -110,6 +111,28 @@ def get_self_link(project, name):
     return f"https://www.googleapis.com/compute/v1/projects/{project}/global/sslCertificates/{name}"
 
 
+def find_cert(project_id, credentials, cert, cert_self_links):
```

Comment:
Are we able to add a test for this new method?

Reply:
Outside of the time-consuming process of writing test data I think it should be possible to mock the response of the GCP SDK to make unit testing this possible, something like below should in theory work as a starting point:
```python
from import google.cloud.compute_v1.types.compute import SslCertificate

@patch("google.cloud.compute_v1.services.ssl_certificates.SslCertificatesClient.get")
def test_find_cert(mock_get_cert):
	mock_get_cert.side_effect = [
		SslCertificate(self_link="<fakelink1>", certificate="<fakecert1>"),
		SslCertificate(self_link="<fakelink2>", certificate="<fakecert2>"),
	] # TODO(populate test data)
	...
```